Lil-Peep/src/styles/modules/site-footer.scss

```diff
@@ -0,0 +1,234 @@
+footer {
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  background-color: #fff;
+  margin-top: 15px;
+  border-top-style: solid;
+  border-top-color: rgba(28, 26, 26, 0.1);
+  border-top-width: 1px;
+}
+
+.footer-content {
+  margin: 0 40px;
+
+  .social-media-container {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    width: 100%;
+
+    .social-media-box {
+      display: flex;
+      flex: 1;
+      text-align: center;
+
+      .box-sm {
+
+        .social-media {
+          color: #72808C;
+          font-family: 'Lato', sans-serif;
+          font-size: 18px;
+          line-height: 21px;
+          display: block;
+
+          p {
+            @media screen and (max-width: 768px) {
```

Comment:
I highly recommend creating variables that store the values for those breakpoints...
This is recommendable when if for some reason those values change then automatically all the references to it will to the same.
Something like this:
```
$extrasmall-phone: 350px;
$small-phone: 480px;
$large-phone: 640px;
$small-tablet: 800px;
$large-tablet: 960px;
$small-laptop: 1120px;
$medium-laptop: 1280px;
$large-laptop: 1440px;
$small-desktop: 1000px;
$medium-desktop: 1760px;
$large-desktop: 1920px;
```

Reply:
You right!, it would be easier :)